en/subscription_services_privacy_notice.md

```diff
@@ -1,55 +1,67 @@
 # Mozilla Subscription Services | Privacy Notice
 
-Version 1.1, Effective January 10, 2024
-{: datetime="2024-01-10" }
+Version 1.2, Effective February 5, 2024
+{: datetime="2024-02-05" }
 
 ## At Mozilla, we design products with your privacy in mind.
 
 __Mozilla VPN__ protects your device’s internet connections. Mozilla partners with Mullvad to privately and securely encrypt your internet traffic.
 
-__Firefox Relay__ protects your identity with secure email and phone masking. Our secure and easy to use email and phone masks help keep your identity private so you can sign up for new accounts anonymously, stop spam texts and junk calls, and get only the emails you want in your inbox.  
+__Mozilla Monitor__ can monitor to see if any of your accounts have been in known data breaches. In some regions, it can also help you scan for and remove your information from places around the web. 
 
-__Firefox Monitor__ can monitor to see if any of your accounts have been in known data breaches. See [Firefox Monitor Terms and Privacy](https://www.mozilla.org/privacy/firefox-monitor/).
+__Firefox Relay__ protects your identity with secure email and phone masking. Our secure and easy to use email and phone masks help keep your identity private so you can sign up for new accounts anonymously, stop spam texts and junk calls, and get only the emails you want in your inbox.  
 
-This privacy notice explains what data Firefox Relay and Mozilla VPN collect, share, and why. We also adhere to the [Mozilla Privacy Policy](https://www.mozilla.org/privacy/) for how we receive, handle, and share information.
+This privacy notice explains what data Mozilla VPN, Mozilla Monitor and Firefox Relay collect, share, and why. We also adhere to the [Mozilla Privacy Policy](https://www.mozilla.org/privacy/) for how we receive, handle, and share information.
 
 ## Things You Should Know
 
 __Mozilla account information.__ These Services require a Mozilla account, which sends Mozilla your email address, locale, and IP address. Learn more about [Mozilla Account data practices](https://www.mozilla.org/privacy/mozilla-accounts/).
 
-__Payment information.__ When you subscribe to the Services, you will send payment through one of our third-party payment providers: Stripe, Apple, PayPal, or Google Pay. Mozilla receives a record of your account (including your billing address and the last four digits of your payment method) and the status of your account’s subscription. Mozilla does not store your full payment details.
+__Payment information.__ When you subscribe to Services that require a paid subscription, you will send payment through one of our third-party payment providers: Stripe, Apple, PayPal, or Google Pay. Mozilla receives a record of your account (including your billing address and the last four digits of your payment method) and the status of your account’s subscription. Mozilla does not store your full payment details.
```

Comment:
Are Mozilla provided subscription services referred to as `Services` (upper case)?

Reply:
Thanks for picking up on this. Agreed that we should just use "services" unless we need to specify "Mozilla Subscription Services" for some reason. I'll update and resubmit.